### PR TITLE
update(css-declaration-sorter): v6 update

### PR DESCRIPTION
**Pull request by peterblazejewicz**
Uses new v8 of `postcss`.

https://github.com/Siilwyn/css-declaration-sorter/commit/e74c93d0eb4ac94c70651040b1c0bfeea5586b2c

Thanks!

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x ] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/css-declaration-sorter/tsconfig.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es6",
+            "es2018"
```
Comment (amcasey) [inline types/css-declaration-sorter/tsconfig.json:6]: "es2018" doesn't imply "es6"?

File: types/css-declaration-sorter/css-declaration-sorter-tests.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import postcss = require('postcss');
+import postcss from 'postcss';
```
Comment (amcasey) [inline types/css-declaration-sorter/css-declaration-sorter-tests.ts:1]: Out of curiosity, why change this?

**Review by amcasey — APPROVED** [review]

File: types/css-declaration-sorter/css-declaration-sorter-tests.ts (hunk 2 above)
Comment (peterblazejewicz) [reply types/css-declaration-sorter/css-declaration-sorter-tests.ts:1]: `postcss` uses now `export default` to support both ESM and JS. This results in `This expression is not callable.` here.

**Comment by peterblazejewicz** [comment]
Ready to merge 💘

**Pull request merged**